easybuild/easyconfigs/p/Pysam/Pysam-0.10.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,51 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+# 0.9.1.4:
+# Modified by: Robert Qiao <robert.qiao@adelaide.edu.au>, Exequiel Sepulveda <exequiel.sepulveda@gmail.com>
+# @ adelaide.edu.au/phoenix
+
+easyblock = 'Bundle'
+
+name = 'Pysam'
+version = '0.10.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/pysam-developers/pysam'
+description = """Pysam is a python module for reading and manipulating Samfiles. 
+ It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+parallel = 1
+
+dependencies = [
+    ('Python', '2.7.12'),
+    ('Cython', '0.25.2', versionsuffix),
+    ('ncurses', '6.0'),
+    ('zlib', '1.2.8'),  
+    ('cURL', '7.49.1'),  
+]
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+exts_list = [
+    ('pysam', version, {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/pysam-developers/pysam/archive/'],
+    }),
+]
+
+modextrapaths = {
+    'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages'],
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/pysam-%(version)s-py%(pyshortver)s-linux-x86_64.egg'],
```

Comment:
this is quite specific, and may cause trouble on some system, we usually just check for `lib/python%(pyshortver)s/site-packages`, which suffices since it's combined with the `import` check done by EasyBuild